test/e2e_node/node_shutdown_linux_test.go

```diff
@@ -59,6 +59,17 @@ var _ = SIGDescribe("GracefulNodeShutdown [Serial] [NodeAlphaFeature:GracefulNod
 			initialConfig.ShutdownGracePeriodCriticalPods = metav1.Duration{Duration: nodeShutdownGracePeriodCriticalPods}
 		})
 
+		ginkgo.BeforeEach(func() {
+			ginkgo.By("Waiting node ready")
```

Comment:
Looks good, but what do you think about just using `waitForNodeReady` from [here](https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/e2e_node_suite_test.go#L264-L283) instead? it seems to already have this logic built in :)

Reply:
That is a nice point, +1